pkg/component/istio/charts/istio/istio-istiod/templates/peerauthentication.yaml

```diff
@@ -1,8 +0,0 @@
-apiVersion: security.istio.io/v1beta1
-kind: PeerAuthentication
-metadata:
-  name: default
-  namespace: {{ .Release.Namespace }}
-spec:
-  mtls:
-    mode: STRICT
```

Comment:
What does the removal of the peer authentication imply? What did it do in the past?